### DISCUSSION TITLE
Paying in instalments

### DISCUSSION THREAD

**Discussion by yazfield** [juspay/hyperswitch#5834] [Ideas / Feature Requests]
I saw this project and found it really nice, i thought it would be cool if it handled instalment payments.
sorry i'm not familiar with the project, can someone add custom code to hyperswitch do add instalments feature if it won't be supported? i'd appreciate if you could give some dirtions.
Thanks!

**Comment by neerajkumar-n** [comment]
@yazfield - are you referring to Subscription payments here? If yes, we do support them using Kill bIll.
Plugin repo: https://github.com/juspay/hyperswitch-killbill-plugin

let me know if this answers your questions.

**Reply by yazfield** [reply]
No instalment payment like pay in 3 times. It's a bit similar to subscription but not quite it. A product costs 90, you buy it now and pay 30 then 30 every month until you have paid the full 90.

**Reply by tinu-hareesswar** [reply]
@yazfield - Hyperswitch currently supports installment usecases by the following ways:

1. Integrations with Buy Now Pay Later providers like Klarna, Affirm, AfterPay etc. for triggering installment payments
2. Tokenization: You can charge the customer 30 initially and tokenize the customer's card during the purchase. Later you initiate charges of 30 every month on the payment method with MITs. 
3. Partial Capture: You can get the customer's authorization for the full product cost of 90 during the purchase, but charge 30 every month on the disbursement of services.

**Reply by yazfield** [reply]
Thanks @tinu-hareesswar for the info 🙏
